topic02-schedule/book-schedule/img/04.December.md

```diff
@@ -0,0 +1,14 @@
+# Schedule for December
+
+## Week 8: Website Review
```

Comment:
Just need to change this to week 11, other than that content looks good. 

File needs to be moved up a directory to student-help-guide-content/topic02-schedule/book-schedule/04.December.md by the way (it's currently in the img directory)

Reply:
thanks guys will fix.